### ISSUE TITLE
Cannot handle large tables (more than 2 billion rows)

### ISSUE THREAD

**Issue by andrewsali** [janmotl/linkifier#3]
It appears that the number of rows stored by linkifier is an Integer. This seems to cause an issue for large tables: 
```
com.mysql.jdbc.exceptions.jdbc4.MySQLDataException: '4.0053988E9' in column '2' is outside valid range for the datatype INTEGER.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:926)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:897)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:886)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:860)
	at com.mysql.jdbc.ResultSetImpl.throwRangeException(ResultSetImpl.java:7058)
	at com.mysql.jdbc.ResultSetImpl.parseIntAsDouble(ResultSetImpl.java:6336)
	at com.mysql.jdbc.ResultSetImpl.getInt(ResultSetImpl.java:2506)
	at vendor.MySQL.useTables(MySQL.java:75)
	at vendor.MySQL.getTableStatistics(MySQL.java:47)
	at main.Schema.getPrimaryKeys(Schema.java:28)
	at example.Linkifier.estimatePk(Linkifier.java:45)
	at controller.Events$RunService$1.call(Events.java:236)
	at controller.Events$RunService$1.call(Events.java:226)
	at javafx.concurrent.Task$TaskCallable.call(Task.java:1423)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at javafx.concurrent.Service.lambda$null$492(Service.java:725)
	at java.security.AccessController.doPrivileged(Native Method)
	at javafx.concurrent.Service.lambda$executeTask$493(Service.java:724)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by janmotl** [comment]
Thank you for a clear bug report. The count of rows is now stored in long variable. The fix was released in version 3.1.8.

**Comment by andrewsali** [comment]
Thanks for the quick fix!
